image/descriptor.go

```diff
@@ -32,6 +33,10 @@ type descriptor struct {
 	Size      int64  `json:"size"`
 }
 
+func (d *descriptor) normalizeDigest() string {
+	return strings.Replace(d.Digest, ":", "-", -1)
+}
```

Comment:
On Fri, Jul 22, 2016 at 07:18:32AM -0700, Antonio Murdaca wrote:
> > +func (d *descriptor) normalizeDigest() string {
> > +	return strings.Replace(d.Digest, ":", "-", -1)
> > +}
>
> I see, but `oci-image-tool` isn't working at all w/o this fix and
> your #159 it's a rather bigger PR with a broader scope, I'd love to
> have this merged so people can use this tool instead of waiting on
> #159

I'm not sure what's holding #159 up, it's been out there for a while
;).  So I'm not sure what timeframe to expect before people can use it
(although folks are certainly free to use it now, without waiting on
the branch to land in this repo ;).